components/lib/cascadeselect/CascadeSelectBase.js

```diff
@@ -75,11 +76,6 @@ const styles = `
         min-width: 100%;
     }
     
-    .p-cascadeselect-panel {
-        position: absolute;
-        top: 0;
-        left: 0;
-    }
```

Comment:
These changes probably mean we need SASS changes for the PrimeTEk themes: https://github.com/primefaces/primereact-sass-theme

Reply:
I checked [_cascadeselect.scss](https://github.com/primefaces/primereact-sass-theme/blob/main/theme-base/components/input/_cascadeselect.scss) and  [themes](https://github.com/primefaces/primereact-sass-theme/tree/main/themes), there are no position related css